src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,56 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+};
+
+export const MovieCard: React.FC<Props> = ({
+  title, description, imgUrl, imdbUrl,
+}) => (
+  <>
+    <div className="card-image">
```

Comment:
```suggestion
  <div className="card-image">
```
React fragment is redundant here. Also, it looks like you missed div for card
![image](https://user-images.githubusercontent.com/96409563/225091232-5996b069-4345-4958-9e07-38e11f588620.png)